blockchain/block/block.go

```diff
@@ -88,37 +79,21 @@ func (b *Block) Deserialize(buf []byte) error {
 	b.Receipts = nil
 
 	// verify merkle root can match after deserialize
-	txHash, err := b.CalculateTxRoot()
-	if err != nil {
-		log.L().Debug("error in getting hash ", zap.Error(err))
-		return err
-	}
-	if err := b.VerifyTxRoot(txHash); err != nil {
+	if err := b.VerifyTxRoot(); err != nil {
 		return err
 	}
 	return nil
 }
 
 // VerifyTxRoot verifies the transaction root hash
-func (b *Block) VerifyTxRoot(root hash.Hash256) error {
-	if b.Header.txRoot != root {
-		return ErrTxRootMismatch
-	}
-	return nil
-}
-
-// VerifyDeltaStateDigest verifies the delta state digest in header
-func (b *Block) VerifyDeltaStateDigest(digest hash.Hash256) error {
-	if b.Header.deltaStateDigest != digest {
-		return ErrDeltaStateMismatch
+func (b *Block) VerifyTxRoot() error {
+	root, err := b.CalculateTxRoot()
+	if err != nil {
+		log.L().Debug("error in getting hash ", zap.Error(err))
```

Comment:
delete extra suffix space

Reply:
this func also used in `blockstore.go`, update there too